src/main/resources/schema.sql

```diff
@@ -0,0 +1,124 @@
+-- public.binary_contents definition
+
+-- Drop table
+
+-- DROP TABLE public.binary_contents;
+
+CREATE TABLE public.binary_contents (
+	id uuid DEFAULT uuid_generate_v4() NOT NULL,
+	created_at timestamp DEFAULT CURRENT_TIMESTAMP NOT NULL,
+	file_name varchar(255) NOT NULL,
+	"size" int8 NOT NULL,
+	content_type varchar(100) NOT NULL,
+	bytes bytea NOT NULL,
+	CONSTRAINT binary_contents_pkey PRIMARY KEY (id)
+);
+
+
+-- public.channels definition
+
+-- Drop table
+
+-- DROP TABLE public.channels;
+
+CREATE TABLE public.channels (
+	id uuid DEFAULT uuid_generate_v4() NOT NULL,
+	created_at timestamp DEFAULT CURRENT_TIMESTAMP NOT NULL,
+	updated_at timestamp NULL,
+	"name" varchar(100) NULL,
+	description varchar(500) NULL,
+	"type" varchar(10) NOT NULL,
+	CONSTRAINT channels_pkey PRIMARY KEY (id)
+);
+
+
+-- public.users definition
+
+-- Drop table
+
+-- DROP TABLE public.users;
+
+CREATE TABLE public.users (
+	id uuid DEFAULT uuid_generate_v4() NOT NULL,
+	created_at timestamp DEFAULT CURRENT_TIMESTAMP NOT NULL,
+	updated_at timestamp NULL,
+	username varchar(50) NOT NULL,
+	email varchar(100) NOT NULL,
+	"password" varchar(60) NOT NULL,
+	profile_id uuid NULL,
+	CONSTRAINT users_email_key UNIQUE (email),
+	CONSTRAINT users_pkey PRIMARY KEY (id),
+	CONSTRAINT users_username_key UNIQUE (username),
+	CONSTRAINT fk_profile_id FOREIGN KEY (profile_id) REFERENCES public.binary_contents(id) ON DELETE CASCADE
```

Comment:
지금 가진 외래키 제약조건에 의하면 binary_contents가 삭제될 때 user도 같이 삭제될 것 같은데요,  
아래와 같이 수정이 필요할 것 같아요.  
```suggestion
	CONSTRAINT fk_profile_id FOREIGN KEY (profile_id) REFERENCES public.binary_contents(id). ON DELETE SET NULL
```

Reply:
아앗 넵!!